### PR TITLE
[3685] Ensure setting the radioBehavior of a group does not affect tabGroupBehavior

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: engine/src/exec-interface-group.cpp [hunk 1]
```diff
@@ -139,8 +139,6 @@ void MCGroup::SetRadioBehavior(MCExecContext& ctxt, uint32_t part, bool setting)
 	bool t_dirty;
 	t_dirty = changeflag(setting, F_RADIO_BEHAVIOR);
 
-	if (flags & F_RADIO_BEHAVIOR)
```
Comment (montegoulding) [inline engine/src/exec-interface-group.cpp:142]: @livecodepanos I'm wondering why this was ever here. Odd thing is that group default is radioBehavior true with tabGroupBehavior false so why would we be forcing tabGroupBehavior true when setting radioBehavior!

**Comment by montegoulding** [comment]
@livecode-vulcan review ok a5e0f7a

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok a5e0f7a6c2df0055a8a172703a0e7d3e3554ea3a

**Comment by livecode-vulcan** [comment]
😞 test failure a5e0f7a6c2df0055a8a172703a0e7d3e3554ea3a

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/244)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/236)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/236)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/236)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/236)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1758)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/140)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/140)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1789)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1779)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1778)
* try-community-x86-win32: [pending](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1813)
* try-community-x86_64-win32: [failure](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1753)

**Comment by livecodepanos** [comment]
@livecode-vulcan retry a5e0f7a

**Comment by livecode-vulcan** [comment]
😎 test success a5e0f7a6c2df0055a8a172703a0e7d3e3554ea3a

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/249)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/241)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/241)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/241)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/241)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1763)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/145)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/145)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1794)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1784)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1783)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1818)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1758)

**Pull request merged**